<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMemberView</class>
 <widget class="QWidget" name="AddMemberView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="viewBackground" stretch="2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="memberInfoLayout" stretch="1,0,0">
       <item>
        <layout class="QGridLayout" name="inputsLayout">
         <item row="2" column="1">
          <widget class="QLineEdit" name="passwordLineEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="addressLabel">
           <property name="text">
            <string>Address</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="phoneLineEdit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="usernameLabel">
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="passwordLabel">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="usernameLineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="emailLabel">
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="phoneLabel">
           <property name="text">
            <string>Phone</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="emailLineEdit"/>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="addressLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="errorOutputLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Add Member</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
